tests/integration_tests/test_sampler.py

```diff
@@ -0,0 +1,160 @@
+from mock import patch
+import pytest
+
+import optuna
+from optuna.integration import CmaEsSampler
+from optuna.integration import SkoptSampler
+from optuna.samplers import BaseSampler  # NOQA
+from optuna.testing.sampler import FirstTrialOnlyRandomSampler
+
+if optuna.types.TYPE_CHECKING:
+    from typing import Callable  # NOQA
+
+parametrize_sampler = pytest.mark.parametrize('sampler_class', [
+    optuna.integration.SkoptSampler,
+    optuna.integration.CmaEsSampler,
+])
+
+
+@pytest.mark.parametrize('sampler_class', [
+    lambda: SkoptSampler(
+        independent_sampler=FirstTrialOnlyRandomSampler(), skopt_kwargs={'n_initial_points': 5}),
+    lambda: CmaEsSampler(independent_sampler=FirstTrialOnlyRandomSampler()),
+])
+def test_suggested_value(sampler_class):
```

Comment:
The tests `test_suggested_value`, `test_sample_independent` and `test_warn_independent_sampling` seem almost the same with ones defined in [test_skopt.py](https://github.com/pfnet/optuna/blob/a9788937563de47be79a34a4c875c952f03d319f/tests/integration_tests/test_skopt.py).
Could you delete the corresponding test functions from `test_skopt.py` to prevent redundant code?

Reply:
You're right. I removed them in https://github.com/pfnet/optuna/pull/447/commits/ba991d5af98b7424620ece2c7edd814fcd48a27f.